### ISSUE TITLE
Is this a virus?

### ISSUE THREAD

**Issue by yeahyeahboblox** [Zytope/Slayer-Leecher-V0.6#1]

**Comment by Clapps-lmao** [comment]
yes, its a RAT (Remote Administration Tool) that allows someone else to backdoor into your computer and do what they want with your computer as they please.

![image](https://user-images.githubusercontent.com/90117687/143662222-9b6dfe16-0619-4a12-bd02-ee510ca05c0c.png)

**Comment by yeahyeahboblox** [comment]
thanks bro, where i cand find the real one?
